File: arrays/2. Median of two sorted arrays/ans.py
def findMedianSortedArrays(nums1,nums2):
    merged_arr = nums1 + nums2
    sorted_arr = sorted(merged_arr)

    length=len(sorted_arr)

    median=0
    if(length%2==0):
        median= ((sorted_arr[(length // 2) - 1] + sorted_arr[length // 2]) / 2)
    else:
        median = (sorted_arr[((length + 1) // 2) - 1])

    return median

print(findMedianSortedArrays([1,2],[3,5]))

File: arrays/2. Median of two sorted arrays/ans.js
const findMedianSortedArrays = (nums1, nums2) => {
    const mergedArr = (nums1.concat(nums2));
    const sorted = mergedArr.sort((a, b) => a - b)
    const len = sorted.length;

    let median;
    if (len % 2 == 0) {
        // consider len is even = 10 [(10/2)-1])+[10/2]) / 2
        median = ((sorted[(len / 2) - 1] + sorted[len / 2]) / 2)
    } else {
        // consider len is odd = 9 ( [((9+1)/2)-1] ) = ans
        median = (sorted[((len + 1) / 2) - 1])
    }
    return median;
}

console.log(findMedianSortedArrays([1, 2], [3,4]));